### PR TITLE
some-ui-fixes

### PR DESCRIPTION
**Pull request by suryansh-egov**
## Choose the appropriate template for your PR:

- [Feature PR](./PULL_REQUEST_TEMPLATE/feature_template.md)
- [Bugfix PR](./PULL_REQUEST_TEMPLATE/bugfix_template.md)
- [Release PR](./PULL_REQUEST_TEMPLATE/release_template.md)

## Release Notes

- **New Features**
	- Enhanced visual hierarchy with new CSS classes for improved styling in various components.
	
- **Bug Fixes**
	- Adjusted error handling logic in the CampaignName component for better user feedback.
	- Improved toast notification display duration in the GeoPode component.

- **Improvements**
	- Updated stylesheet link for health UI components to the latest version.
	- Refined button dimensions in the BoundaryHome component for better layout.
	- Streamlined styling approach by replacing inline styles with CSS classes in multiple components.

- **Chores**
	- Updated import statements for consistency across components.

### CODE REVIEW

File: health/micro-ui/web/micro-ui-internals/packages/css/src/pages/employee/campaign.scss [hunk 1]
```diff
@@ -19,11 +19,13 @@
   align-items: start;
 }
 .campaign-type {
```
[the remark this answers is not in the document]
Comment (Bhavya-egov) [reply health/micro-ui/web/micro-ui-internals/packages/css/src/pages/employee/campaign.scss:21]: yes

**Review by Bhavya-egov — APPROVED** [review]

File: health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/CampaignType.js [hunk 2]
```diff
@@ -110,7 +110,7 @@ const CampaignSelection = ({ onSelect, formData, formState, ...props }) => {
           <Card className = "setup-campaign-card">
             <Header styles={{ margin: '0px' }}>{t(`HCM_CAMPAIGN_TYPE_HEADER`)}</Header>
             <p className="description-type">{t(`HCM_CAMPAIGN_TYPE_DESCRIPTION`)}</p>
-            <LabelFieldPair style={{ alignItems: "center" , display: "flex" }}>
+            <LabelFieldPair style={{ display: "flex", marginBottom:"-0.8rem" }}>
```
[the remark this answers is not in the document]
Comment (suryansh-egov) [reply health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/CampaignType.js:113]: solves the issue.

File: health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/GeoPode.js [hunk 3]
```diff
@@ -112,7 +112,7 @@ const GeoPode = () => {
 
         try {
             // setShowLoader(true);
-            setShowToast({ label: "HIERARCHY_PLEASE_WAIT", isError: "info", transitionTime:100});
+            setShowToast({ label: "HIERARCHY_PLEASE_WAIT", isError: "info", transitionTime:100000});
```
[the remark this answers is not in the document]
Comment (suryansh-egov) [reply health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/GeoPode.js:115]: it is in millisecond and gets overwritten when success.

File: health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewHierarchy.js [hunk 4]
```diff
@@ -358,7 +358,7 @@ const ViewHierarchy = () => {
                 {firstPage && 
                     <div>
                         <Card type={"primary"} variant={"viewcard"} className={"example-view-card"}>
-                            <div style={{ fontSize: "2.5rem", fontWeight: 700 }}>{t(`HIERARCHY`)} {hierarchyType}</div>
+                            <div className="hierarchy-boundary-heading">{t(`HIERARCHY`)} {hierarchyType}</div>
```
[the remark this answers is not in the document]
Comment (suryansh-egov) [reply health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewHierarchy.js:361]: the hierarchy name will be show as it is.

**Pull request merged**